fetch-publication/src/test/resources/publication.json

```diff
@@ -1,111 +1,114 @@
 {
-  "type": "Publication",
-  "createdDate": "2020-03-05T12:52:15.887Z",
-  "status": "Draft",
-  "handle": "http://example.org/handle/123",
-  "modifiedDate": "2020-03-05T12:52:15.887Z",
-  "owner": "eier@example.org",
-  "identifier": "9623fee3-49b4-4334-a923-9ee4129dca91",
-  "link": "http://example.org/link",
-  "publisher": {
-    "type": "Organization",
-    "id": "http://example.org/org/123",
-    "labels": {
-      "no": "Eksempelforlaget"
+  "type" : "Publication",
+  "identifier" : "730efffe-e033-4f38-8e3f-7a41515b26d4",
+  "status" : "Draft",
+  "owner" : "eier@example.org",
+  "publisher" : {
+    "type" : "Organization",
+    "id" : "http://example.org/org/123",
+    "labels" : {
+      "no" : "Eksempelforlaget"
     }
   },
-  "entityDescription": {
-    "type": "EntityDescription",
-    "publicationType": "JournalArticle",
-    "mainTitle": "Hovedtittelen",
-    "alternativeTitles": {
-      "en": "English title"
+  "createdDate" : "2020-03-30T05:51:02.487Z",
+  "modifiedDate" : "2020-03-30T05:51:02.487Z",
+  "handle" : "http://example.org/handle/123",
+  "link" : "http://example.org/link",
+  "entityDescription" : {
+    "type" : "EntityDescription",
+    "publicationType" : "JournalArticle",
+    "mainTitle" : "Hovedtittelen",
+    "alternativeTitles" : {
+      "en" : "English title"
     },
-    "language": "http://example.org/norsk",
-    "date": {
-      "type": "PublicationDate",
-      "year": "2020",
-      "month": "4",
-      "day": "7"
+    "language" : "http://example.org/norsk",
+    "date" : {
+      "type" : "PublicationDate",
+      "year" : "2020",
+      "month" : "4",
+      "day" : "7"
     },
-    "contributors": [
-      {
-        "type": "Contributor",
-        "identity": {
-          "type": "Identity",
-          "id": "http://example.org/person/123",
-          "name": "Navnesen, Navn",
-          "nameType": "Personal",
-          "orcId": "orc123",
-          "arpId": "arp123"
-        },
-        "affiliations": [
-          {
-            "type": "Organization",
-            "id": "http://example.org/org/123",
-            "labels": {
-              "no": "Eksempelforlaget"
-            }
-          }
-        ],
-        "role": "Creator",
-        "sequence": 0
-      }
-    ]
+    "contributors" : [ {
+      "type" : "Contributor",
+      "identity" : {
+        "type" : "Identity",
+        "id" : "http://example.org/person/123",
+        "name" : "Navnesen, Navn",
+        "nameType" : "Personal",
+        "orcId" : "orc123",
```

Comment:
General comment, I feel that these should always be URIs and that we can probably do away with the naming of them — they're additional identifiers and we have a preference for ids from ARP and then ORCID and then "others".

Reply:
Create a task to update datamodel?